3-Column.html

```diff
@@ -83,13 +83,13 @@
                         <div id="access">
                             <ul>
                                 <li aria-label="Keyboard Tab Skip">
-                                    <a accesskey="0" href="#main-content-anchor" aria-label="Skip to main content">Skip to main content</a>
+                                    <a href="#main-content-anchor" aria-label="Skip to main content">Skip to main content</a>

```

Comment:
I'm unclear why this `aria-label` is needed since it says the same thing as the content of the `<a>` tag.